pom.xml

```diff
@@ -132,6 +127,16 @@
                 <artifactId>siddhi-execution-list</artifactId>
                 <version>${siddhi.execution.list}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>${commons.io.version}</version>
+            </dependency>
+            <dependency>
+                <groupId> org.apache.cassandra</groupId>
+                <artifactId>cassandra-all</artifactId>
+                <version>0.8.1</version>
```

Comment:
@Ajanthy lets get the versions in the `properties` section

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/82/commits/bea46527dba085b2c5bcc206f49786b72192804e